qgrad/qgrad_qutip.py

```diff
@@ -506,3 +507,14 @@ def __call__(self, thetas, phis, omegas):
                 # (i-1, j-1) to match numpy matrix indexing
                 param_idx += 1
         return jnp.dot(diagonal, rotation)
+
+def rand_unitary(N):
+    r"""Returns an :math:`N \times N` randomly parameterized unitary"""
```

Comment:
The N has to be documented as the Hilbert space size. Explain briefly what the PRNGKey does. Also add the keyword arg `seed` which you can use to initialise your random number generator. I would suggest to use the same seed and generate all the three angles in one go and then take the first set for thetas, second set for phis and third set for omegas

```
def rand_unitary(N, seed=None):
    """
    """
    if seed == None:
        seed = np.random.randint(1000)
    params = uniform(PRNGKey(seed), (N**2),), 
                                    minval=0.0, maxval=2 * jnp.pi)
    rand_thetas = params[:N*(N-1)//2]
    rand_phis = params[N*(N-1)//2:N*(N-1)]
    rand_omegas = params[N*(N-1):]
    ...
```